themes/dark_spinel.json

```diff
@@ -0,0 +1,363 @@
+{
+  "name": "Spinel",
+  "colors": {
```

Comment:
Are the colour choices based on an existing theme?

Reply:
They're not. Why?